theano/gof/op.py

```diff
@@ -694,6 +694,9 @@ def __ne__(self, other):
     # Python implementation #
     #########################
 
+    def grad2(self, inputs, outputs, output_grads):
```

Comment:
This issue won't block this PR as it isn't documented for now. But if we merge it like this, we need an issue about it and hopefully we should find a good fix not too long. I would like a fix before 0.9 to don't release new interface that we could change later.